codetools/mypy/mypy.ini

```diff
@@ -41,3 +38,6 @@ ignore_missing_imports = True
 
 [mypy-setuptools.*]
 ignore_missing_imports = True
+
+[mypy-revolve2.simulation.*]
```

Comment:
what's up with this? why is this here

Reply:
The CI fails for some reason at mypy tests, wince it cant resolve the subs for this package